CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ version directory, and  then changes are introduced.
 - Make --image-pull-progress-deadline configurable for kubelets so a longer
 duration can be used in AWS China regions to mitigate slow image pulls.
 - Harden `restricted` podsecuritypolicy.
+- Update kubernetes to 1.14.5 CVE-2019-1002101, CVE-2019-11246
```

Comment:
Do we need this since we moved in `4.5.1` already? Do we also need a new WIP because azure is using this version now?

Reply:
Oh right it is included there in the 4.5.1 so not much sense imo. thank you for noticing this.